Index: Elastic search/RangeQuery
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>range query => used price field to check wheather amount under lte=30 and gte=25\n\nquery=>\n\n\ncurl -X GET \"localhost:9200/my_index/_search\" -H 'Content-Type:application/json' -d '{\n\"query\":{\n\"range\":{\n\"price\":{\n\"gte\":25,\n\"lte\":30}\n}}}'\n\n\nresult=>\n\n\"took\":6,\"timed_out\":false,\"_shards\":{\"total\":5,\"successful\":5,\"skipped\":0,\"failed\":0},\n\"hits\":{\"total\":2,\"max_score\":1.0,\"hits\":[{\"_index\":\"my_index\",\"_type\":\"_doc\",\"_id\":\"2\",\"_score\":1.0,\"_source\":{\n\"title\":\"Journey of elastic search Advaced\",\n\"author\":\"prekrishna\",\n\"publiation_date\":\"2024-12-22\",\n\"price\":25.89\n}},{\"_index\":\"my_index\",\"_type\":\"_doc\",\"_id\":\"1\",\"_score\":1.0,\"_source\":\n{\n  \"title\": \"Elasticsearch for Beginners\",\n  \"author\": \"John Doe\",\n  \"publication_date\": \"2023-01-01\",\n  \"price\": 29.99\n}\n}]}}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Elastic search/RangeQuery b/Elastic search/RangeQuery
--- a/Elastic search/RangeQuery	(revision f3582ebf97d8c2de8ede8a4e166fad687ffcb716)
+++ b/Elastic search/RangeQuery	(date 1737875917832)
@@ -12,6 +12,8 @@
 }}}'
 
 
+
+
 result=>
 
 "took":6,"timed_out":false,"_shards":{"total":5,"successful":5,"skipped":0,"failed":0},
